azext_iot/monitor/property.py

```diff
@@ -90,34 +92,31 @@ def _changed_props(self, prop, metadata, property_name):
         }
         return diff
 
+    def _is_component(self, prop):
+        return type(prop) == dict and prop.get(COMPONENT_MARKER) == "c"
+
     def _validate_payload(self, changes, minimum_severity):
         for value in changes:
-            issues = self._validate_payload_against_interfaces(
+            issues = self._validate_payload_against_interfaces_components(
```

Comment:
i liked your earlier name - does naming this `_validate_payload_against_entities` work?

If not, lets just rename it to components for parity with dtdlv2 and remove all references to interfaces outside of user facing strings and dtdlv1 template internal logic

Reply:
_validate_payload_against_entities  works - components and interfaces co-exist - cannot remove interfaces from the reference here.